src/Lykke.Service.QuotesHistory/Controllers/QuotesHistoryController.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+using Lykke.Service.Assets.Client;
+using Lykke.Service.QuotesHistory.Core.Domain.Quotes;
+using Lykke.Service.QuotesHistory.Models;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.SwaggerGen;
+
+namespace Lykke.Service.QuotesHistory.Controllers
+{
+    [Route("api/[controller]")]
+    [Produces("application/json")]
+    public sealed class QuotesHistoryController : Controller
+    {
+        private readonly IQuoteHistoryRepository _quoteHistoryRepository;
+        private readonly IAssetsService _assetsService;
```

Comment:
`IAssetsServiceWithCache` will be better here. You can use `RegisterAssetsClient()` extension method of the `IServiceCollection` to register `IAssetsServiceWithCache` in the DI container. Thus to use this extension method, you need `IServiceCollection` in your `ServiceModule`. You can do it as in the template - https://github.com/LykkeCity/lykke.dotnettemplates/blob/master/Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Modules/ServiceModule.cs#L17